Graphs/Table/index.js

```diff
@@ -117,10 +117,8 @@ class Table extends AbstractGraph {
     static getRemovedColumns(columns, filterColumns, selectedColumns) {
         let removedColumns = [];
         columns.forEach((d, index) => {
-            if (filterColumns.length) {
-                if (!filterColumns.find(column => d.column === column))
-                    removedColumns.push(`${index}`)
-
+            if (d.displayOption && filterColumns.length && !filterColumns.find(column => d.column === column)) {
```

Comment:
you should keep the inner if and just add to it d.displayOption otherwise you completely change the logic afterwards! You might have filterColumns.length but not d.displayOption!

Reply:
there are 3 conditions under which a column can be removed. we want all 3 conditions to be tested.